activerecord/test/cases/tasks/sqlite_rake_test.rb

```diff
@@ -15,6 +15,9 @@ def setup
       File.stubs(:exist?).returns(false)
       ActiveRecord::Base.stubs(:connection).returns(@connection)
       ActiveRecord::Base.stubs(:establish_connection).returns(true)
+
+      @original_stdout = $stdout
```

Comment:
I can't see any use of the `@original_stdout` instance variable.
Should we asset on this variable instead of `$stdout.expects(:puts)` as @jeremy mentioned?